src/ListTabulator/index.ts

```diff
@@ -268,6 +235,92 @@ export default class ListTabulator {
     };
   }
 
+  /**
+   * Method that specified hot to merge two List blocks.
+   * Called by Editor.js by backspace at the beginning of the Block
+   *
+   * Content of the first item of the next List would be merged with deepest item in current list
+   * Other items of the next List would be appended to the current list without any changes in nesting levels
+   *
+   * @param {ListData} data - data of the second list to be merged with current
+   * @public
+   */
+  merge(data: ListData): void {
+    console.log('holder', this.block.holder);
+
+    /**
+     * Get list of all levels children of the previous item
+     */
+    const items = this.block.holder.querySelectorAll<ItemElement>(`.${DefaultListCssClasses.item}`);
```

Comment:
can you use `getChildItems()` here?

Reply:
i responded above, here we get all item elements of the list (all nesting levels), not only `childItems` of the one level